geographies_list.py

```diff
@@ -21,11 +21,15 @@
     reader = csv.reader(file)
     code_country_list = list(reader)
     code_country_dict = {}
+    country_code_dict = {}
     for row in code_country_list:
-        # key is city (row[0]), value is country (row[1])
-        city_country_dict[row[1]] = row[0]
+        # key is country code (row[1]), value is country (row[0])
+        code_country_dict[row[1]] = row[0]
+        # key is country (row[0]), value is country code (row[1])
+        country_code_dict[row[0]] = row[1]
 
-CODE_COUNTRY_DICT = city_country_dict
+CODE_COUNTRY_DICT = code_country_dict
```

Comment:
@kdobolyi, can you please add a simple comment explaining why there are two similarly named dicts? I know it's obvious, but it took me--a donut, to borrow your phrase--a minute to figure out what is going on.